pkg/fmmon/alert.go

```diff
@@ -0,0 +1,36 @@
+package fmmon
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+// Alert sends the specified message to the specified webhook URL.
+func Alert(msg string, webhookURL string) {
+	m := `{"text":"` + msg + `"}`
```

Comment:
Use `encoding/json.Marshal` instead. This will cause breakage if `msg` contains a `"` or `\n`.